### PR TITLE
Fix for lifecycle postStart and preStop tests - excepting cnf-app-mac-operator

### PR DESCRIPTION
**Pull request by ramperher**
cnf-app-mac-operator case will be addressed in a separate change.

### CODE REVIEW

**Comment by ramperher** [comment]
> * SUCCESS https://www.distributed-ci.io/jobs/37815ba2-52a1-4813-ac7a-42b9f0f15676/jobStates
>   from [[WIP] Fix for lifecycle postStart and preStop tests #34](https://github.com/openshift-kni/example-cnf/pull/34)

In this change, you can see the lifecycle-container-startup/shutdown tests are now passing, and the rest of tests remain in the same status, so this is ready for review.

**Comment by ramperher** [comment]
Let's wait for confirmation from tnf regarding the issue we have with cnf-app-mac-operator, just to know if it's a hard requirement to have a shell for running commands.

**Comment by ramperher** [comment]
I've decided not to change cnf-app-mac-operator in this PR, I'll address it in a separate change. The changes for the other operators are correct and this can be merged. @capolrik @tkrishtop could anyone of you please take a look to the change to have it merged?

File: testpmd-lb-operator/config/manager/manager.yaml [hunk 1]
```diff
@@ -31,6 +31,13 @@ spec:
             - name: ANSIBLE_GATHERING
               value: explicit
           image: controller:latest
+          lifecycle:
+            postStart:
+              exec:
+                command: ["/bin/sh", "-c", "echo Hello from the postStart handler"]
+            preStop:
+              exec:
+                command: ["/bin/sh", "-c", "echo Hello from the preStop handler"]
```
Comment (tkrishtop) [inline testpmd-lb-operator/config/manager/manager.yaml:40]: are the requirements to have logs or to simply have lifecycle/postStart? 

```
        lifecycle:
          postStart: {}
          preStop: {}
```

Comment (ramperher) [reply testpmd-lb-operator/config/manager/manager.yaml:40]: Good observation. Probably, according to the [tnf code](https://github.com/test-network-function/cnf-certification-test/blob/main/cnf-certification-test/lifecycle/suite.go#L259-L290), the test can be passed by just defining an empty postStart and preStop block, but I believe it's better to have an example of postStart and preStop to demonstrate how it works, even if it's just an `echo`.

**Comment by ramperher** [comment]
This change will be used to integrate the fix for cnf-app-mac-operator: https://github.com/openshift-kni/example-cnf/pull/35.

In the meanwhile, this PR can be merged safely.

File: testpmd-lb-operator/config/manager/manager.yaml (hunk 1 above)
Comment (tkrishtop) [reply testpmd-lb-operator/config/manager/manager.yaml:40]: I think we're good then, let's merge

**Review by tkrishtop — APPROVED** [review]

**Pull request merged**